.github/workflows/ci.yml

```
@@ -133,6 +133,18 @@ jobs:
           cd examples
           make -f MakefileExternal LIBTRIXI_PREFIX=$PWD/../install
 
+      - name: Test building in external CMake project
+        if: ${{ matrix.test_type == 'regular' }}
+        run: |
+          mkdir examples/test_cmake
+          cp examples/simple_trixi_controller.c examples/test_cmake/
+          cp examples/CMakeListsExternal.txt    examples/test_cmake/CMakeLists.txt
+          cp cmake/FindLibTrixi.cmake           examples/test_cmake/
+          mkdir examples/test_cmake/build
+          cd    examples/test_cmake/build
+          cmake -DLIBTRIXI_PREFIX=$PWD/../../../install ..
+          make
```

```suggestion
          mkdir examples/external/build
          cd    examples/external/build
          cmake -DLIBTRIXI_PREFIX=$PWD/../../../install ..
          make
```
Better yet: create a shell script `build.sh` that contains this code and put it in `examples/external`. This way the user sees exactly what needs to be executed and the code is checked during CI at the same time.